### PR TITLE
Add gthread and gaiohttp to settings docs

### PR DESCRIPTION
**Pull request by strife25**
Added the worker class types of `gthread` and `gaiohttp` to the settings docs.

### CODE REVIEW

File: docs/source/settings.rst [hunk 1]
```diff
@@ -102,6 +102,8 @@ A string referring to one of the following bundled classes:
 * ``eventlet`` - Requires eventlet >= 0.9.7
 * ``gevent``   - Requires gevent >= 0.13
 * ``tornado``  - Requires tornado >= 0.2
+* ``gthread``  - Requires Python 3
+* ``gaiohttp``  - Requires Python 3 and aiohttp >= 0.21.5
```
Comment (strife25) [inline docs/source/settings.rst]: I am not sure what minimum versions should be mentioned here or if there are other dependencies that are needed. 

Let me know if and how I should fix these.

Comment (benoitc) [inline docs/source/settings.rst]: gthread can actually works with python 2 once the futures are installed. Other than that it's OK :)

File: docs/source/settings.rst [hunk 2]
```diff
@@ -102,6 +102,8 @@ A string referring to one of the following bundled classes:
 * ``eventlet`` - Requires eventlet >= 0.9.7
 * ``gevent``   - Requires gevent >= 0.13
 * ``tornado``  - Requires tornado >= 0.2
+* ``gthread``  - Requires Python 3
```
Comment (berkerpeksag) [inline docs/source/settings.rst]: gthread requires the "futures" package to be installed in Python 2.

**Comment by berkerpeksag** [comment]
Merged in https://github.com/benoitc/gunicorn/commit/6a3bd70d2b6a8961f8bb1c16df58b4d7a3e83836, thanks! :)

**Pull request closed without merging**